articles/dms/tutorial-mysql-azure-mysql-offline-portal.md

```diff
@@ -0,0 +1,254 @@
+---
+title: "Tutorial: Migrate MySQL offline to Azure Database for MySQL"
+titleSuffix: "Azure Database Migration Service"
+description: "Learn to perform an offline migration from MySQL on-premises to Azure Database for MySQL by using Azure Database Migration Service."
+services: dms
+author: "sumitgaurin"
+ms.author: "sgaur"
+manager: "arthiaga"
+ms.reviewer: "arthiaga"
+ms.service: dms
+ms.workload: data-services
+ms.custom: "seo-lt-2019"
+ms.topic: tutorial
+ms.date: 04/11/2021
+---
+
+# Tutorial: Migrate MySQL to Azure Database for MySQL offline using DMS
+
+You can use Azure Database Migration Service to perform a one-time full database migration on-premises MySQL instance to [Azure Database for MySQL](../mysql/index.yml) with high speed data migration capability. In this tutorial, we will migrate a sample database from an on-premises instance of MySQL 5.7 to Azure Database for MySQL (v5.7) by using an offline migration activity in Azure Database Migration Service. Although the articles assumes the source to be a MySQL database instance and target to be Azure Database for MySQL, it can be used to migrate from one Azure Database for MySQL to another just by changing the source server name and credentials. Also, migration from lower version MySQL servers (v5.6 and above) to higher versions is also supported.
+
+> [!IMPORTANT]
+> The “MySQL to Azure Database for MySQL” online migration scenario is being replaced with a parallelized, highly performant offline migration scenario from June 1, 2021. For online migrations, you can use this new offering together with [data-in replication](https://docs.microsoft.com/azure/mysql/concepts-data-in-replication). Alternatively, use open-source tools such as [MyDumper/MyLoader](https://centminmod.com/mydumper.html) with data-in replication for online migrations. 
+
+> [!INCLUDE [preview features callout](../../includes/dms-boilerplate-preview.md)]
+
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+>
+> * Migrate database schema using mysqldump utility.
+> * Create an instance of Azure Database Migration Service.
+> * Create a migration project by using Azure Database Migration Service.
+> * Run the migration.
+> * Monitor the migration.
+
+## Prerequisites
+
+To complete this tutorial, you need to:
+
+* Have an Azure account with an active subscription. [Create an account for free] (https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+* Have an on-premises MySQL database with version 5.7 or 8. If not, then download and install [MySQL community edition](https://dev.mysql.com/downloads/mysql/) 5.7 or 8.
```

Comment:
We also support 5.6

Reply:
Updated to 
"Have an on-premises MySQL database with version 5.7. If not, then download and install [MySQL community edition](https://dev.mysql.com/downloads/mysql/) 5.7."
This is because the introduction paragraph we have mentioned : "In this tutorial, we will migrate a sample database from an on-premises instance of MySQL 5.7 to Azure Database for MySQL (v5.7) by using an offline migration activity in Azure Database Migration Service."

Reply:
In introduction we have mentioned - "Also, migration from lower version MySQL servers (v5.6 and above) to higher versions is also supported."